wscript

```diff
@@ -456,6 +456,10 @@ def configure(cfg):
     if cfg.options.num_aux_imus > 0:
         cfg.define('INS_AUX_INSTANCES', cfg.options.num_aux_imus)
 
+    # require python 3.8.x or later
+    cfg.load('python')
+    cfg.check_python_version(minver=(3,8,0))
```

Comment:
```suggestion
    cfg.check_python_version(minver=(3,6,0))
```

Let Ubuntu 18.04 continue to function

Reply:
yep, 3.8 don't bring much functionnality against 3.6, so using 3.6 is good

Reply:
> yep, 3.8 don't bring much functionnality against 3.6, so using 3.6 is good

Discussed this with tridge.

bionic came with 3.8 and it's almost defunct.

Python 3.6 is *also* out of support.